src/main/scala/tech/cryptonomic/conseil/config/ConseilAppConfig.scala

```diff
@@ -0,0 +1,28 @@
+package tech.cryptonomic.conseil.config
+
+import tech.cryptonomic.conseil.config.Platforms._
+import pureconfig.generic.auto._
+
+
+/** wraps all configuration needed to run Conseil */
+trait ConseilAppConfig {
```

Comment:
Out of curiosity, why are these traits and not just regular behaviours of Conseil, Lorre, etc? Will multiple classes ever inherit these traits?

Reply:
The simple answer is "separation of concerns" and "flexibility".

I chose one between many possible configuration parsing libraries for scala (the choice in this regard is as rich almost as with json-parsing).
Keeping the trait separated allows to relatively restrain the amount of code directly dependent on the _pureconfig_ external library.
The goal is to keep the specifics of implementation grouped in as few possible sources as possible... the same reasoning applies to how `ConfigUtils` keep the code related to pureconfig separated from the rest of the codebase (by keeping it in a custom util file) and from other possible utilities (by enclosing that code in a specific sub-object)